topics/web/serializers/example4/main.go

```diff
@@ -54,11 +54,11 @@ func main() {
 		log.Fatal(err)
 	}
 
-	print("User with data", b)
+	printMsgData("User with data", b)
 
 	fmt.Println("\nNote in all examples 'email' is missing.")
 }
 
-func print(msg string, data []byte) {
+func printMsgData(msg string, data []byte) {
```

Comment:
This is probably a good change so people don't confuse `print` with a builtin function.

Reply:
Agree